app/serializers/basic_post_serializer.rb

```diff
@@ -46,7 +46,6 @@ def cooked
   def ignored
```

Comment:
Should this return false if the site setting is not enabled? Seems like a fast way to exit without doing much work.

Reply:
Yes, good point.